marimbabot_vision/scripts/flatten_whiteboard_data.py

```diff
@@ -0,0 +1,25 @@
+import os
+import tqdm
+from multiprocessing import Pool
+import re
+from shutil import copyfile
+
+NUM_WORKER = 30
+INPUT_DIR = 'data'
+OUTPUT_DIR = 'data_wb'
+NAME_PREFIX = 'staff_1'
+
+def main(sample):
```

Comment:
I would recommend to add a comment on what this does and maybe why its needed

Reply:
Yes, you are absolutely right. I added a description to argparse, hope that will do.